src/main/java/com/github/bpiatek/bbghbackend/ninetyminutes/domain/NinetyMinutesPlayerCrawler.java

```diff
@@ -0,0 +1,52 @@
+package com.github.bpiatek.bbghbackend.ninetyminutes.domain;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+
+import com.github.bpiatek.bbghbackend.model.player.Player;
+import com.github.bpiatek.bbghbackend.model.player.PlayerFacade;
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.jsoup.Jsoup;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+/**
+ * Created by Bartosz Piatek on 12/10/2020
+ */
+@Log4j2
+@Service
+@AllArgsConstructor
+class NinetyMinutesPlayerCrawler {
+
+  private final NinetyMinutesPlayerExtractor playerExtractor;
+  private final PlayerFacade playerFacade;
+
+  void startCrawlingForPlayers() {
```

Comment:
Tyle jest na chwile obecna profili na portalu wiec zrobilem najlatwiej jak mozna i wbilem to na sztywno :)